test/parallel/test-crypto-prime.js

```diff
@@ -295,3 +297,17 @@ assert.throws(() => {
     checkPrime(prime, common.mustSucceed(assert));
   }));
 }
+
+{
+  // Verify that generatePrime can be reasonably interrupted.
+  const worker = new Worker(`
+    const { generatePrime } = require('crypto');
+    generatePrime(2048, () => {
+      throw new Error('should not be called');
+    });
+    process.exit(0);
+  `, { eval: true });
+
+  worker.on('error', common.mustNotCall());
+  worker.on('exit', common.mustCall());
```

Comment:
nit 
```suggestion
  worker.on('exit', common.mustCall((exitCode) => assert.strictEqual(exitCode, 0)));
```
It might make sense to use 42 as exit code as we sometimes do in tests